views/overlays/bracket.hbs

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>{{title}}</title>
+    <script src="/socket.io/socket.io.js"></script>
+  </head>
+  <body>
+    <script>
+    var socket = io('/challonge');
+    var embedString = '/module?multiplier=0.8&theme=2&match_width_multiplier=0.8';
+
+    socket.on('update challonge', function(data) {
+      var url = data.challongeUrl || '';
```

Comment:
This doesn't have to be `OR`ed, it'll still return false in the following if statement if it's undefined.